### PR TITLE
Add support for setting agent labels

### PR DESCRIPTION
**Pull request by colinking**
This will allow folks to add their own agent labels or override our existing labels.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -13,6 +13,11 @@ variable "team_id" {
   type = string
 }
 
+variable "labels" {
```
Comment (joshma) [inline variables.tf:16]: Let's make this optional? List of strings (each one is `"key:value"`), default to empty list, join with `" "` in the template?

File: packer/conf/bin/configure-airplane.sh [hunk 2]
```diff
@@ -10,7 +10,7 @@ AP_AGENT_NAME=${INSTANCE_ID}-%s
 AP_API_HOST=${AP_API_HOST}
 AP_API_TOKEN=${AP_API_TOKEN}
 AP_TEAM_ID=${AP_TEAM_ID}
-AP_LABELS=aws_instance_id:${INSTANCE_ID} aws_region:${AWS_REGION} aws_account_id:${AWS_ACCOUNT_ID}
+AP_LABELS=cloud:aws aws_instance_id:${INSTANCE_ID} aws_region:${AWS_REGION} aws_account_id:${AWS_ACCOUNT_ID} ${AP_LABELS}
```
Comment (joshma) [inline packer/conf/bin/configure-airplane.sh:13]: Since this is in the packer image, we have to rebuild it and update the template w/ the new AMIs.

**Review by joshma — APPROVED** [review]

File: variables.tf (hunk 1 above)
Comment (colinking) [reply variables.tf:16]: Didn't realize I needed a default to be optional, thanks! And I like that, but used a map instead. Feels more intuitive to me and ordering doesn't matter here afaik.

Comment (joshma) [reply variables.tf:16]: ooh nice

**Pull request merged**